### DISCUSSION TITLE
NextJS Image placeholder Blur still shows after page was first viewed

### DISCUSSION THREAD

**Discussion by JakeHolcroft** [vercel/next.js#52924] [Help] [answered]
### Summary

NextJS Image placeholder Blur still shows after page was first viewed

I am using getStaticProps to statically generate the pages and using wordpress as my CMS, I am using 100% width and height images so they can take a while to load so i automatically generated a blur hash for each image so that i can use the next placeholder blur.

I would expect that on the first load, there is the blur and then after visiting the page again the image would then appear instantly which is the case if i remove the blur; but with the blur on it uses the blur hash every time

**Comment by LimSB-dev** [comment, marked as answer]
It seems like you are experiencing a caching issue with Next.js Image placeholder blur. By default, Next.js caches the generated blur hash for each image on the client-side to improve performance. This means that even after the first load, the blur hash will still be used for subsequent visits, causing the blur effect to persist.

To have the image appear instantly after the first load and bypass the blur hash for subsequent visits, you can use the `priority` attribute for the `<Image>` component. The `priority` attribute tells Next.js to prioritize loading the image, which means it will load the image immediately instead of showing the placeholder blur.

Here's how you can modify your code to achieve this:

1. Make sure you have the `Image` component from Next.js imported:

```javascript
import Image from 'next/image';
```

2. Use the `priority` attribute in your `<Image>` component:

```javascript
<Image
  src="/path/to/your/image.jpg"
  alt="Alt Text"
  width={1000}
  height={600}
  layout="responsive"
  placeholder="blur"
  blurDataURL="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mP8/w95HgAGggKtPNcAAAAASUVORK5CYII="
  priority // Add this attribute to load the image immediately
/>
```

By adding the `priority` attribute, the image will be fetched immediately, and the blur hash won't be used on subsequent visits, resulting in the image appearing instantly after the first load.

Remember to re-deploy your application after making these changes to ensure the modifications take effect in the production environment.

**Comment by JakeHolcroft** [comment]
Thanks a lot! @LimSB-dev 

I was using priority={true} but this mustn't have been working, I replaced it with "priority" only and it works.

Maybe the next image docs need updating.
https://nextjs.org/docs/pages/api-reference/components/image#priority

Thanks
Jake
